src/sui/themes/rtd-site/elements/button.overrides

```diff
@@ -0,0 +1,113 @@
+.ui.button {
+  vertical-align: middle;// Required for .button to be aligned with .button.animated
+}
+
+
+.ui.basic.buttons form > .button {
+  &:last-child {
+    border-top-right-radius: 0rem;
+    border-bottom-right-radius: 0rem;
+  }
+
+  &:hover {
+    box-shadow: none;
+  }
+
+  border-left: 0rem;
+}
+
+/*******************************
+     Gradient Variants (new)
+*******************************/
+
+/*-------------------
+      Secondary
+      Gradient
+--------------------*/
+
+.ui.secondary.buttons.gradient .button,
+.ui.secondary.button.gradient {
+  background-image: @secondaryGradientBackgroundImage;
+}
+.ui.secondary.buttons.gradient .button:hover,
+.ui.secondary.button.gradient:hover {
+  background-image: @secondaryGradientHoverBackgroundImage;
+}
+.ui.secondary.buttons .button:focus,
+.ui.secondary.button:focus {
+  background-image: @secondaryGradientFocusBackgroundImage;
+}
+.ui.secondary.buttons .active.button,
+.ui.secondary.buttons .active.button:active,
+.ui.secondary.active.button,
+.ui.secondary.button .active.button:active {
+  background-image: @secondaryGradientActiveBackgroundImage;
+}
+
+
+/*******************************
+     Outline Variants (new)
+*******************************/
+/* 
+  TODO1: Override .basic buttons with these styles and remove the .outline class.
+  TODO2: Add the full list of .basic buttons to our /examples/theme/index.html.
+*/
+
+.ui.outline.button {
+  background: transparent none;
+  text-shadow: none !important;
+  transition: box-shadow @defaultDuration;
+}
+/* Mixin for .outline states */
+.buttonOutlineState(@color) {
+  box-shadow: 0px 0px 0px 1px @color inset !important;
+  color: @color;
+
+  &:hover,
+  &:focus,
+  &.active,
+  &.active:focus {
+    box-shadow: 0px 0px 0px 2px @color inset !important;
+    background: none !important;
+  }
+}
+
+/* States */
+.ui.outline.button {
+
+  &.black     { .buttonOutlineState(@black) }
+  &.primary   { .buttonOutlineState(@primaryColor) }
+  &.secondary { .buttonOutlineState(@secondaryColor) }
+  &.blue      { .buttonOutlineState(@primaryColor) }
+  &.green      { .buttonOutlineState(@green) }
+
+}
+
+/*******************************
+        Size Variants
+*******************************/
+.ui.tiny.button {
+  padding: (@tinyVerticalPadding + .05) @tinyHorizontalPadding @tinyVerticalPadding;
+  font-weight: @normal;
+}
+.ui.mini.button {
+  padding: (@miniVerticalPadding + .05) @miniHorizontalPadding @miniVerticalPadding;
+  font-weight: @normal;
+}
```

Comment:
Good point. I'll add a comment.

In the meantime: it's required because SUI sizes the buttons using only `font-size` which I think is really short sighted for a framework offering a "mini" button that has really small text but huge padding comparatively.

I ended up not using the `.mini.button` in the homepage but I believe it's handy to have style and that we'll be using it later.

Reply:
Ah yes, I have hit this. `ui compact mini button` is probably closer to what you want https://semantic-ui.com/elements/button.html#compact

Reply:
It's closer!
I'll remove this code for now as I ended up not using it.

I bet we'll still need some overrides to this style later as, for the smallest button we have available, it's still to big --imagine an action button inside a table with 50 rows and a button on every row. We'll want a button as small as it can ergonomically go.